src/Cms/App.php

```diff
@@ -725,9 +717,9 @@ public static function instance(self $instance = null, bool $lazy = false)
 		if ($instance === null) {
 			if ($lazy === true) {
 				return static::$instance;
-			} else {
-				return static::$instance ?? new static();
 			}
+
+			return static::$instance ?? new static();
 		}
 
 		return static::$instance = $instance;
```

Comment:
We could get rid of the nested if here by inverting the logic?

```
       if ($instance !== null) {
            return static::$instance = $instance;
        }

        if ($lazy === true) {
            return static::$instance;
        }

        return static::$instance ?? new static();
```

Reply:
Yes!